nlds/rabbit_connection.py

```diff
@@ -59,10 +62,10 @@ def create_message(transaction_id: uuid, action: str, contents: str) -> str:
 
         return f"{time}:{transaction_id}:{action}:[{contents}]"
 
-    def publish_message(self, msg: str) -> None:
+    def publish_message(self, routing_key: str, msg: str) -> None:
```

Comment:
Again, what happens if we fail to publish a message.
We need some logging in here, I think.
And maybe using the @retry decorator.

Reply:
Did you mean a specific package (i.e. https://pypi.org/project/retry/) or make our own?

Reply:
I meant the retry package.

Reply:
Will be addressed in #6